openstudiocore/src/cli/EmbeddedHelp.hpp

```diff
@@ -7,36 +7,21 @@
 #include <windows.h>
 #endif
 
-extern "C" {
```

Comment:
Can't exactly tell what you're up to but i think you're just removing the shenanigans to delay init this json stuff, and instead you're just doing it on os load. Fine by me. I like the cleanup. This always seems yucky to me.